app/webpacker/components/RegistrationsV2/RegistrationEdit/RegistrationEditor.jsx

```diff
@@ -67,7 +63,7 @@ export default function RegistrationEditor({ competitor, competitionInfo }) {
     },
     onSuccess: (data) => {
       queryClient.setQueryData(
-        ['registration-admin', competitionInfo.id, competitor.id],
+        ['registration', competitionInfo.id, competitor.id],
```

Comment:
Will this cause problems if you're "meddling" with the query state inside the context?

Reply:
It would probably be better if the context/hook was doing the mutation internally, and providing `updateRegistrationMutation` and `isUpdating` to this component.

Reply:
I tried this out, but I need to call `nextStep` in the `onSuccess`, and I can't do that if I move it to the Context.